ledger-api/grpc-definitions/com/daml/ledger/api/v1/event.proto

```diff
@@ -68,11 +68,18 @@ message CreatedEvent {
   // Optional
   Record create_arguments = 4;
 
-  // Opaque representation of contract payload intended for forwarding
+  // Opaque representation of contract create argument data intended for forwarding
   // to an API server as a contract disclosed as part of a command
   // submission.
+  // Deprecated in favor of ``contract_create_payload``.
   // Optional
-  google.protobuf.Any create_arguments_blob = 12;
+  google.protobuf.Any create_arguments_blob = 12 [deprecated = true];
+
+  // Opaque representation of complete contract create payload intended for forwarding
```

Comment:
I think data is more general than payload (payload might alias more closely to create args in people's minds no?)

```suggestion
  // Opaque representation of complete contract create data intended for forwarding
```

Reply:
Ok, I see that it was create payload and suggested to change to `data`. I won't bikeshed on this if it was already discussed

Reply:
I have changed this formulation following @meiersi-da comment